_alerts/2017-10-09-segwit2x-safety.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Beware of Bitcoin's incompatibility with Coinbase & others"
+shorturl: "segwit2x-safety"
+active: true
+banner: "Beware of Bitcoin's incompatibility with Coinbase & others"
+bannerclass: "alert"
+date: 2017-10-09
+---
+
+## Summary
+
+A subsection of the Bitcoin economy including Coinbase have signed an agreement to adopt
+and implement a contentious hard fork of Bitcoin [sometime](http://bashco.github.io/2x_Countdown/) in November. A hard fork is a
+backwards incompatible change to the Bitcoin network. This hard fork is not supported by
+the majority of the Bitcoin users and developers and is therefore a contentious hard fork.
```

Comment:
A majority would not be sufficient to avoid being a contentious hard fork.  This should be reworded to make sure even a majority cannot hard fork bitcoin unilaterally.

Reply:
What about "This hard fork is not supported by an overwhelming majority..."?